packages/components/src/custom-select-control-v2/stories/default.story.tsx

```diff
@@ -11,11 +12,12 @@ import { useState } from '@wordpress/element';
 /**
  * Internal dependencies
  */
+import NewCustomSelect from '../default-component';
 import { CustomSelect, CustomSelectItem } from '..';
 
-const meta: Meta< typeof CustomSelect > = {
-	title: 'Components (Experimental)/CustomSelectControl v2',
-	component: CustomSelect,
+const meta: Meta< typeof NewCustomSelect > = {
```

Comment:
The change from `typeof CustomSelect` to `typeof NewCustomSelect` should be also applied later in the file where we use `StoryFn< ... >`.

That should help remove the `//@ts-nocheck` annotation.

Reply:
Good catch! The no-check was added to run the storybook build, and I forgot to remove it. Otherwise it would be pointing these things out to me. 😅 Thank you!